test_env.py

```diff
@@ -43,12 +43,15 @@ def test_numpy():
         os_numpy_version = os.environ['NUMPY_VERSION'].lower()
         if 'dev' in os_numpy_version:
             assert 'dev' in numpy.__version__
+        elif 'pre' in os_numpy_version:
+            assert 'pre' in numpy.__version__
         else:
             if 'stable' in os_numpy_version:
                 assert numpy.__version__.startswith(LATEST_NUMPY_STABLE)
             else:
                 assert numpy.__version__.startswith(os_numpy_version)
             assert 'dev' not in numpy.__version__
+            assert 'pre' not in numpy.__version__
```

Comment:
This isn't right, the version numbering you get for pre-releases is like ``x.y.z.a1/b2/rc3``. See here for a regexp that may be of use: https://github.com/numpy/numpy/blob/master/numpy/tests/test_numpy_version.py